dev/io.openliberty.cdi.4.1.internal_fat/build.gradle

```diff
@@ -0,0 +1,14 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+addRequiredLibraries.dependsOn addJakartaTransformer
```

Comment:
This file should be able to be removed.  We aren't doing the transformer with this test since it already uses jakarta classes.

Reply:
Hmm, I think the one I copied it from needed it because the repeat code calls the transformer, but this one doesn't have any repeats (yet) so it shouldn't need it.